docs/source/api/tutorials/In Depth 3 - Drawing DiHypergraphs.ipynb

```diff
@@ -13,7 +13,7 @@
    "id": "ea8aad6a",
```

Comment:
<p>The output was <code>(ax, node_coll, edge_coll)</code> and is now <code>(ax, node_coll, edge_coll, dyad_coll) </code>if I got that right. But the dyad_coll is a list of <code>FancyArrowPatch</code> and not an actual python <code>Collection</code>. </p><p> </p><p>The reason to output the collection was (i) for easy colorbars and (ii) to easily change parameters of the whole collection (like alpha value). What use did you have in mind for outputting the new list? Maybe we could make it an actual collection instead?</p><p> </p><p>I think my initial thought for not needing to output is that I thought either the arrows would be multicolored but consistent with the edges, in which case the edge_coll is sufficient, or be a single neutral color like black, in which case maybe returning them is less useful.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/492/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Yea I was thinking maybe we don't return the list of patches. In any case I dont see a case where we would need a different colorbar for edge markers and bipartite edges. Maybe we enforce that if bipartite edges need to be colored with a cmap, it needs to be the same as that of edges markers?</p>